snap/hooks/configure

```diff
@@ -48,6 +48,7 @@ if [[ -d $SNAP_COMMON/lxd ]]; then
 fi
 
 if [[ -f $SNAP_COMMON/lxd/unix.socket ]]; then
+    groupadd lxd || true
```

Comment:
Why are we only creating the group if the socket file already exists?  I would expect that if the socket file exists, there's a good chance that the group already exists anyway.  It seems like a fresh install is the case where we'd actually need to create the group, so this seems like it should be outside the `if`.